lib/models/conversation.dart

```diff
@@ -14,7 +14,6 @@ class Conversation implements Model {
   @override
   Map<String, dynamic> toMap() {
     return {
-      'id': id,
```

Comment:
why remove  the id? what happens if we have a group chat? we cant base it off of a userid only